pkg/http/api_crypto_test.go

```diff
@@ -96,25 +96,6 @@ func TestCryptoEndpoints(t *testing.T) {
 		}
 	})
 
-	t.Run("Missing component name - 400", func(t *testing.T) {
```

Comment:
With the updated router this case is impossible because `//` gets simplified to `/`. In any case, this will still cause an error, just a not specific one.